.github/workflows/nightly-standalone-snapshot.yml

```diff
@@ -0,0 +1,59 @@
+name: Nightly Test run (Snapshot)
```

Comment:
Same comment than above regarding the name of the workflow.

`Nightly (Jahia Snapshot)`

Reply:
Fixed